<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QK7P72A" failures="1" tests="12" name="com.api.tests.GenericCRM" time="16.519" errors="0" timestamp="2 Mar 2020 13:46:43 GMT">
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveOptionSet" time="0.646"/>
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveCounties" time="0.653"/>
  <testcase classname="com.api.tests.GenericCRM" name="GetContactRolesByContactId" time="0.769"/>
  <testcase classname="com.api.tests.GenericCRM" name="PayTransaction" time="0.900">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.api.utils.VerifyResponse.validateAssertion(VerifyResponse.java:21)
	at com.api.tests.GenericCRM.PayTransaction(GenericCRM.java:303)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
    </failure>
  </testcase> <!-- PayTransaction -->
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveEntities" time="4.075"/>
  <testcase classname="com.api.tests.GenericCRM" name="GetAllContactRoles" time="1.528"/>
  <testcase classname="com.api.tests.GenericCRM" name="LogMemberValue" time="2.052"/>
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveAllBranches" time="0.593"/>
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveMultiplesOptionsSet" time="1.222"/>
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveCountries" time="0.698"/>
  <testcase classname="com.api.tests.GenericCRM" name="CreateContactActivatedAndValidated" time="2.607"/>
  <testcase classname="com.api.tests.GenericCRM" name="RetrieveAllPeriods" time="0.776"/>
</testsuite> <!-- com.api.tests.GenericCRM -->
